subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -289,10 +287,16 @@ source "subsys/logging/Kconfig.template.log_config"
 
 endif
 
-
 config NRF_CLOUD_CONNECTION_POLL_THREAD
 	bool "Poll cloud connection in a separate thread"
 
+config NRF_CLOUD_MQTT_KEEPALIVE
```

Comment:
I'm not quite sure that we need this option, as `CONFIG_MQTT_KEEPALIVE` is used in the existing applications today.  Is there a use case where using `CONFIG_MQTT_KEEPALIVE` is not working as intended?

Reply:
SLM has its own MQTT service, which does not have to use the same KEEPALIVE as what's configured for nRF Cloud.